<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QWidget" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>861</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
    <weight>9</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>FrmMain</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;Calibri&quot;;</string>
  </property>
  <widget class="QListWidget" name="lwAnzeige">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>791</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSortKontonr">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>sortieren nach KontoNummer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSortName">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>sortieren nach Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Online Banking App&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Konto anlegen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Konto bearbeiten&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNewName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>296</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUpName">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>anlegen!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>1121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ausführen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kontostand:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNewKontoStand">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUpBetrag">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Ein-/Auszahlung&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Überweisung&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUeBetrag">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>740</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>740</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Betrag:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUeberweisung">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>780</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Überweisen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNewlastName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUpLastName">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>170</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Löschen!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nachname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUpNameNeu">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>490</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUpLastNameNeu">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nachname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>460</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Neu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIBAN3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>700</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>27</number>
   </property>
   <property name="placeholderText">
    <string>DE09 0900 1900 0900 1900 91</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>iBAN:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIBAN2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>27</number>
   </property>
   <property name="placeholderText">
    <string extracomment="DE09 0900 1900 0900 1900 91">DE09 0900 1900 0900 1900 91</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IBAN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIBAN4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>580</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>27</number>
   </property>
   <property name="placeholderText">
    <string extracomment="DE09 0900 1900 0900 1900 91">DE09 0900 1900 0900 1900 91</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIBAN">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>27</number>
   </property>
   <property name="placeholderText">
    <string extracomment="DE09 0900 1900 0900 1900 91">DE09 0900 1900 0900 1900 91</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>iBAN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>740</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwendungszweck:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIBAN5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>700</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>27</number>
   </property>
   <property name="placeholderText">
    <string>DE09 0900 1900 0900 1900 91</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leZweck">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>740</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>700</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Absender IBAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>700</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Empfänger IBAN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIBANneu">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>580</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>27</number>
   </property>
   <property name="placeholderText">
    <string>DE09 0900 1900 0900 1900 91</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Neue IBAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>740</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Betrag:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
